src/main/java/com/github/camotoy/geyserpreventserverswitch/bungeecord/ReloadCommand.java

```diff
@@ -0,0 +1,28 @@
+package com.github.camotoy.geyserpreventserverswitch.bungeecord;
+
+import net.md_5.bungee.api.CommandSender;
+import net.md_5.bungee.api.chat.TextComponent;
+import net.md_5.bungee.api.connection.ProxiedPlayer;
+import net.md_5.bungee.api.plugin.Command;
+
+public class ReloadCommand extends Command {
+
+    private BungeeCordPreventServerSwitch plugin;
+
+    protected ReloadCommand(BungeeCordPreventServerSwitch plugin) {
+        super("pssreload", "preventserverswitch.reload");
```

Comment:
Should probably be `gsspreload` since this is Geyser.

Reply:
geyser-server-switch-prevent-reload?